docker-compose.mongo.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  mongodb:
+    image: bitnami/mongodb:4.2
+    volumes:
+      - mongodb_data:/bitnami
+    ports:
```

Comment:
Leaving the port open to a database can be problematic (our IT department scans for this and will shut off network access).

I generally do one of two things:


If I really need access to mongo from other processes on my machine:
```
ports:
   - 127.0.0.1:27017:27017

```

If I just need to let services from other docker compose files see this, define a network for all docker-compose (this is something required even within a single docker-compose for things like podman on Mac, which we are starting to use.)

Reply:
Yeah, this is only used for the Github action. I can also move this file into the `.github` folder to make it more clear?